apps/explorer/src/components/Object/ScrollToViewCard.tsx

```diff
@@ -1,23 +1,27 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { useEffect, useRef, type ReactNode } from 'react';
+import { useEffect, useRef, type ReactNode, type MutableRefObject } from 'react';
 
 interface ScrollToViewCardProps {
 	children: ReactNode;
 	inView: boolean;
+	containerRef: MutableRefObject<HTMLDivElement | null>;
```

Comment:
Seems confusing to me that this takes a container ref and also has a scroll ref. Seems like only one should be required.

Reply:
remove the need to pass containerRef inside